jupyter/datascience/ubi9-python-3.11/runtime-images/rocm-pytorch-ubi9-py311.json

```diff
@@ -0,0 +1,12 @@
+{
+  "display_name": "Pytorch with ROCm and Python 3.11 (UBI9)",
+  "metadata": {
+    "tags": [
+      "rocm-pytorch"
```

Comment:
I wonder whether we want the tags defined this way or whether we would like to have them as separate instead:
```
[ "rocm", "pytorch" ]
```
:thinking: 

But I guess that these images are build together with ROCm for some reason, so probably having it as one tag makes sense :shrug:

Reply:
Honestly, it would help longer to tags them separate, as tags could be use to filter out runtimes images.
however as we currently are in earlier stages, keeping them as single tag would be more helpful.
![Screenshot from 2024-09-17 15-28-32](https://github.com/user-attachments/assets/eda54509-05dd-4b2a-ab0f-f8b8afff7556)